docs/source/guides/staking_guide.rst

```diff
@@ -53,31 +68,71 @@ Staking Overview
 
 Most stakers on the Goerli testnet will complete the following steps:
 
-1) Install nucypher https://docs.nucypher.com/en/latest/guides/installation_guide.html
-2) Install and run Geth, Parity or another ethereum node.
+1) Install nucypher on StakeHolder node https://docs.nucypher.com/en/latest/guides/installation_guide.html
+2) Install and run Geth, Parity or another ethereum node (can use with software or hardware Ethereum wallet).
 3) Request testnet tokens from faucet
 4) Stake tokens (See Below)
-5) Initialize a Worker node and bond it to your Staker (`set-worker`) https://docs.nucypher.com/en/latest/guides/ursula_configuration_guide.html
-6) Run the Worker, and keep it online!
+5) Install another Ethereum node at the Worker instance
+6) Initialize a Worker node and bond it to your Staker (``set-worker``) :ref:`ursula-config-guide`
+7) Configure and run the Worker, and keep it online (See Below)!
 
 Interactive Method
 ------------------
 
-*Initialize a new stakeholder*
+Run an Ethereum node for stakeholder
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Assuming you have ``geth`` installed, let's run a node on Görli testnet.
```

Comment:
We should be consistent with our spelling - we have used both `Goerli` and `Görli`.

Reply:
Yeah, ok, `oe` is kind of a way to spell when you cannot type `ö`. I assume, can be interchangeable?